### DISCUSSION TITLE
minio cluster start error，The same directory on a node starts normally，error：Unable to use the drive http://xx.xx.xx.xx:9000/opt/data/minio: drive not found

### DISCUSSION THREAD

**Discussion by z624023251** [minio/minio#20739] [Q&A]
minio version：RELEASE.2024-11-07T00-52-20Z

OS version：
![image](https://github.com/user-attachments/assets/927b0017-723d-4f22-87e6-59d5f05dc4f5)

error log：
![image](https://github.com/user-attachments/assets/0bcb374a-ceb0-416a-ac67-666060c90b5b)

journalctl log：
![image](https://github.com/user-attachments/assets/d009f4ab-315e-4018-89db-041c23acb7f7)

systemctl status minio log：
![image](https://github.com/user-attachments/assets/7f2a9b81-b933-4969-9539-2b29750e09c4)

The ProtectProc option is removed and cannot be started. The error message is the same。

Only see so many logs, or where can I see more logs?

**Comment by marktheunissen** [comment]
Please do not create duplicate issues when your previous issue was already converted to a discussion here https://github.com/minio/minio/discussions/20734
